src/parachain/amm.ts

```diff
@@ -783,7 +783,7 @@ export class DefaultAMMAPI implements AMMAPI {
         }
 
         const lpTokenCurrencyId = newCurrencyId(this.api, pool.lpToken);
-        const farmDepositExtrinsic = this.api.tx.farming.deposit(lpTokenCurrencyId);
+        const farmDepositExtrinsic = this.api.tx.farming.deposit(lpTokenCurrencyId, 123);
```

Comment:
@sander2 @gregdhill I assume this was just a quick fix to silent the TSC typecheck. Regarding this change, I noticed that new parameter is `length_rewards: upper bound for number of reward currencies`, how do we need to handle this on lib side?

Reply:
Correct. You should set it to the length of the `farmingRewards.RewardCurrencies(lpTokenCurrencyId)`